plugins/TunePMuonProducer.cc

```diff
@@ -0,0 +1,47 @@
+#include "DataFormats/PatCandidates/interface/Muon.h"
```

Comment:
I am not just selecting some muons, instead I am replacing the muon p4 with the tuneP track information. This way is easier to calculate the masses and other quantities.